build_tools/github_actions/configure_ci.py

```diff
@@ -208,6 +209,8 @@ def get_ci_stage(event_name):
     return "presubmit"
   elif event_name == PUSH_EVENT_NAME:
     return "postsubmit"
+  elif event_name == SCHEDULE_EVENT_NAME:
+    return "schedule"
```

Comment:
Schedule should probably use "postsubmit" as the stage since it runs on already merged code.
```suggestion
    return "postsubmit"
```